sbnana/CAFAna/Systs/NuMIFluxSysts.h

```diff
@@ -11,6 +13,8 @@ namespace ana
 
   class NuMIFluxSyst : public ISyst
   {
+    static constexpr std::string_view fluxFileName = "2023-07-06_out_450.37_7991.98_79512.66_QEL11.root";
```

Comment:
Is there a potential use case here where we could actually allow it to declare a location on construction and make the name and location a combined data member which defaults to this file? Maybe not? But asking...